sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/b2c/AADB2CUserPrincipalTest.java

```diff
@@ -75,11 +75,11 @@ public void testNoArgumentsConstructorDefaultScopeAndRoleAuthorities() {
 
     @Test
     public void testNoArgumentsConstructorExtractScopeAuthorities() {
-        when(jwt.containsClaim("scp")).thenReturn(true);
+        when(jwt.getClaim("roles")).thenReturn(null);
```

Comment:
Same here.

Reply:
Hi, @moarychan , release in June has already finished. please update this PR.

Reply:
Hi @chenrujun , done, please help review.